app/controllers/UserController.py

```diff
@@ -0,0 +1,196 @@
+from app.models.user import User
+from app import response, app, db
+from app.blacklist import blacklist
+from flask import request, jsonify
+import datetime
+from flask_jwt_extended import *
+
+def getAllUsers():
```

Comment:
this function name should be `get_all_users()`

read more about naming function in python:
https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
done